httpx/client.py

```diff
@@ -64,6 +69,7 @@ def __init__(
         headers: HeaderTypes = None,
         cookies: CookieTypes = None,
         timeout: TimeoutTypes = DEFAULT_TIMEOUT_CONFIG,
+        retries: RetriesTypes = DEFAULT_RETRIES_CONFIG,
```

Comment:
Could we perhaps drop `DEFAULT_RETRIES_CONFIG` and instead use `retries: RetriesTypes = None` throughout?

Reply:
Sure yes! Makes sense given that the default behavior is to not retry, and `None` isn't a special value in this case.